ansible/config/monkecar/ds4drv.env

```diff
@@ -0,0 +1 @@
+CONTROLLER_LED_COLOR=FCF003
```

Comment:
Why yellow ? shouldn't it be white which is the car's color ?

Reply:
Because I didn't know if white wasn't already taken by the controller when powered on, so I wanted to make sure that there was a clear distinction between the controller being on, and it being connected to the car.

Reply:
It's white blinking I think we can keep it white

Reply:
fixed in 6d69fa480bcdb7656f6f4098de8a1f9d6dffc0ab